quality/detekt-config-compose.yml

```diff
@@ -0,0 +1,40 @@
+naming:
+  FunctionNaming:
+    functionPattern: '[a-zA-Z][a-zA-Z0-9]*'
+  TopLevelPropertyNaming:
+    constantPattern: '[A-Z][A-Za-z0-9]*'
+
+style:
+  MagicNumber:
+    ignoreNumbers:
+      - '-1'
+      - '0'
+      - '1'
+      - '2'
+      - '3'
+    ignorePropertyDeclaration: true
+    ignoreCompanionObjectPropertyDeclaration: true
+  MaxLineLength:
+    maxLineLength: 100
+    excludeCommentStatements: true
+  ForbiddenComment:
+    allowedPatterns: 'TODO:'
+  WildcardImport:
```

Comment:
I would not use wildcard imports in code at all, if possible. It's a sign of bad style, and it's not good for the code maintenance for many reasons.